src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -880,32 +804,42 @@ def main(argv):
     if use_minijail:
       bind_corpus_dirs(minijail_chroot, [new_testcase_mutations_directory])
 
-  max_len_argument = fuzzer_utils.extract_argument(
-      arguments, constants.MAX_LEN_FLAG, remove=False)
-  if not max_len_argument and do_random_max_length():
-    max_length = random.SystemRandom().randint(1, MAX_VALUE_FOR_MAX_LENGTH)
-    arguments.append('%s%d' % (constants.MAX_LEN_FLAG, max_length))
-    fuzzing_strategies.append(strategy.RANDOM_MAX_LENGTH_STRATEGY)
+  if strategy_pool[strategy.RANDOM_MAX_LENGTH_STRATEGY]:
+    max_len_argument = fuzzer_utils.extract_argument(
+        arguments, constants.MAX_LEN_FLAG, remove=False)
+    if not max_len_argument:
+      max_length = random.SystemRandom().randint(1, MAX_VALUE_FOR_MAX_LENGTH)
+      arguments.append('%s%d' % (constants.MAX_LEN_FLAG, max_length))
+      fuzzing_strategies.append(strategy.RANDOM_MAX_LENGTH_STRATEGY)
 
-  if do_recommended_dictionary():
+  if strategy_pool[strategy.RECOMMENDED_DICTIONARY_STRATEGY]:
     if add_recommended_dictionary(arguments, fuzzer_name, fuzzer_path):
       fuzzing_strategies.append(strategy.RECOMMENDED_DICTIONARY_STRATEGY)
 
-  if do_value_profile():
+  if strategy_pool[strategy.VALUE_PROFILE_STRATEGY]:
     arguments.append(constants.VALUE_PROFILE_ARGUMENT)
     fuzzing_strategies.append(strategy.VALUE_PROFILE_STRATEGY)
 
-  if do_fork():
-    max_fuzz_threads = environment.get_value('MAX_FUZZ_THREADS', 1)
-    num_fuzz_processes = max(1, multiprocessing.cpu_count() // max_fuzz_threads)
-    arguments.append('%s%d' % (constants.FORK_FLAG, num_fuzz_processes))
-    fuzzing_strategies.append(
-        '%s_%d' % (strategy.FORK_STRATEGY, num_fuzz_processes))
+  # TODO(crbug.com/920355): Reenable this when fork mode works with ChromeOS's
```

Comment:
This comment can be removed per my comment below, but FYI this should have probably been placed right before the if memory_tool != 'MSAN'